t/29-openqa.t

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env perl -w
```

Comment:
the name of the file is a bit weird. A test called "openqa"?

Reply:
Well, it covers OpenQA.pm, what do you suggest?

Reply:
ah, ok. Hm, maybe "29-openqa_daemon.t"? Otherwise I could imagine newcomers would be confused which feature this test should test.

Reply:
i actually wanted to put this test in another one (since it's just split in 3 bunch of subtests), but i could not find something more "affine" to it - not even sure of openqa_daemon.t as well, since we do not use that code to run the daemons :/ - that file it's confusing on first place i guess 😕